wd.html

```diff
@@ -739,7 +743,7 @@ <h3>Switching windows or tabs</h3>
     # Wait for the new tab to finish loading content
     wait.until(EC.title_is("Selenium documentation"))</code>
  <code class=cs>//Store the ID of the original window
-String originalWindow = driver.CurrentWindowHandle;
+string originalWindow = driver.CurrentWindowHandle;
```

Comment:
arguable :)

Reply:
I didn't read selenium's source code, but MS's libraries and docs prefer `string` than `String`.

Reply:
Sure